src/msgs/handshake.rs

```diff
@@ -281,8 +281,6 @@ impl SupportedMandatedSignatureSchemes for SupportedSignatureSchemes {
             SignatureScheme::RSA_PKCS1_SHA512,
             SignatureScheme::RSA_PKCS1_SHA384,
             SignatureScheme::RSA_PKCS1_SHA256,
-
-            SignatureScheme::RSA_PKCS1_SHA1,
```

Comment:
:sigh: Sorry I missed this.

IMO, this is a symptom of a bigger problem in Rustls: Rustls defines enum values for every algorithm, cipher suite, TLS version, etc., but only implements a subset of them. AFAICT it would be much better if each enum had its values limited to the set actually implemented by Rustls. Then this bug wouldn't have occurred.